src/service/performance-impl.js

```diff
@@ -301,8 +301,12 @@ export class Performance {
         recordedFirstInputDelay = true;
       } else if (entry.entryType === 'layoutJank') {
         this.aggregateJankScore_ += entry.fraction;
-      } else if (entry.entryType === 'layoutShift') {
-        this.aggregateShiftScore_ += entry.value;
+      } else if (entry.entryType === 'layout-shift') {
```

Comment:
Do you also need `layoutShift` here? Without it,  I think you'll miss entries from Chrome 76.

Reply:
Thank you for catching this - added support for both versions in d569ea7f94acbfd062997def8d880b6351fb5727